Frends.Excel.Parse/Frends.Excel.Parse/Excel.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.ComponentModel;
+using System.Text;
+using ExcelDataReader;
+using Frends.Excel.Parse.Definitions;
+
+namespace Frends.Excel.Parse;
+
+public static class Excel
+{
+    /// <summary>
+    /// Converts Excel file to data set. [Documentation](https://github.com/FrendsPlatform/Frends.Excel/tree/main/Frends.Excel.Parse)
```

Comment:
Please change the documentation link according to latest guidelines.

Reply:
Done.